app/controllers/user.js

```diff
@@ -0,0 +1,22 @@
+const validations = require('../util');
+const userService = require('../services/users');
+const errors = require('../errors');
+const logger = require('../logger');
+
+exports.createUser = (req, res, next) => {
+  const { body } = req;
```

Comment:
Apply destructuring with the things inside req.body.
We use destructuring in order to avoid making object.property1, object.property2, etc many times